node/channel.js

```diff
@@ -255,6 +257,13 @@ function TChannel(options) {
         self.flushStats();
     }
 
+    if (!self.topChannel) {
+        self.errorBackoff = new ErrorBackoff({
```

Comment:
I recommend CongestionAvoidance (AIMD, controlled by errors), to be later complemented by FlowControl (controlled by latency). AIMD is "additive increase, multiplicative decrease". I see the multiplicative decrease part, but no mechanism for an edge to recover from failure. Be sure to trap successes as well as failures to complete the control loop.